content/en/tracing/profiler/enabling/dotnet.md

```diff
@@ -232,6 +232,6 @@ The [Getting Started with Profiler][5] guide takes a sample service with a perfo
 
 [1]: https://app.datadoghq.com/account/settings#agent/overview
 [2]: https://app.datadoghq.com/account/settings?agent_version=6#agent
-[3]: https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.1.1-profiler-beta1
+[3]: https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.5.1
```

Comment:
Since this link can get outdated, what if we just link to the releases page instead of a specific release?